### ISSUE TITLE
Unrecognized option "default_provider" under "bazinga_geocoder"

### ISSUE THREAD

**Issue by bmeynell** [geocoder-php/BazingaGeocoderBundle#100]
`composer install`

>  [Symfony\Component\Config\Definition\Exception\InvalidConfigurationException]  
>    Unrecognized option "default_provider" under "bazinga_geocoder" 

`config.yml`:

``` yml
bazinga_geocoder:                                                                  
    default_provider: google_maps                                                  
    providers:                                                                     
        google_maps:                                                               
            locale:               ~                                                
            region:               ~                                                
            use_ssl:              true                                             
        free_geo_ip: ~                                                             
        geo_plugin: ~
```

`composer.json`:

`"willdurand/geocoder-bundle": "3.1.*@dev",`

This is a very recent error ...

**Comment by willdurand** [comment]
ah :(

**Comment by youuri** [comment]
+1

**Comment by vvh-empora** [comment]
+1

**Comment by bmeynell** [comment]
What happened here? Is the `default_provider` needed any longer? I've been removing from all my projects to avoid this error...

**Comment by Nyholm** [comment]
This will be fixed in #127
